### PR TITLE
chore: build with upload

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -297,6 +298,16 @@ jobs:
...
 
+      - name: tar builds
+        env:
+          ALI_OSS_AK_ID: ${{ secrets.ALI_OSS_AK_ID }}
+          ALI_OSS_AK_SECRET: ${{ secrets.ALI_OSS_AK_SECRET }}
+          HEAD_SHA: ${{ github.sha }}
+        run: |
+          tar -czvf build-artifacts.tar.gz dist locale es lib
+          echo "🤖 Uploading"
+          node scripts/visual-regression/upload.js ./build-artifacts.tar.gz --ref=$HEAD_SHA
```
Comment (crazyair) [inline .github/workflows/test.yml:309]: 都不舍得写个新的脚本

Comment (zombieJ) [reply .github/workflows/test.yml]: 脚本就是 `upload.js`

**Comment by afc163** [comment]
图片

1. 两个有一个能找到构建产物就不应该报错。
2. 另外这个报错信息能 catch 一下再格式化输出么，现在太长了不好读。

**Comment by zombieJ** [comment]
> 1. 两个有一个能找到构建产物就不应该报错。
> 2. 另外这个报错信息能 catch 一下再格式化输出么，现在太长了不好读。

现在就是的，截图里的报错是因为两个产物都没有。。。

**Review by vagusX — DISMISSED** [review]

**Review by afc163 — APPROVED** [review]

**Comment by zombieJ** [comment]
@SocketSecurity ignore npm/esbuild@0.18.20
@SocketSecurity ignore npm/@swc/core@1.4.2
@SocketSecurity ignore npm/@fortawesome/free-regular-svg-icons@6.5.2
@SocketSecurity ignore npm/@fortawesome/free-solid-svg-icons@6.5.2
@SocketSecurity ignore npm/@fortawesome/fontawesome-common-types@6.5.2

**Pull request merged**